CONTRIBUTING.rst

```diff
@@ -0,0 +1,44 @@
+Contributing
+============
+
+Pull requests are the best way to propose changes to the codebase.
+Contributions are welcome, but they have to meet some criteria.
+
+Pull Request Process
+--------------------
+
+1. Fork this Git repository and create your branch from ``master``.
+
+2. Make sure the tests pass:
+
+   .. code:: bash
+
+    python setup.py test
+
+3. If you add code that should be tested, add tests.
+
+4. Make sure the linters pass:
```

Comment:
All these could be replaced by a simple pre-commit run.

Reply:
Same reason as for your other posts: I want these instructions to be simple and understandable for everyone.